ai/unet/Model Training.py

```
@@ -64,54 +94,267 @@
+            train_parts.append(GlaucomaDatasetMergedMask(
+                IMAGES_DIR_B, MASKS_DIR_B,
+                transform=train_transform))
+            val_parts.append(GlaucomaDatasetMergedMask(
+                IMAGES_DIR_B, MASKS_DIR_B,
+                transform=val_transform))
+        else:
+            print(f"[Layout B] Skipped — path not found: {IMAGES_DIR_B}")
+
+    if not train_parts:
+        raise RuntimeError("No dataset could be loaded! "
+                           "Check IMAGES_DIR_A / IMAGES_DIR_B paths.")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)